packages/react-components/src/components/InputBoxComponent.test.tsx

```diff
@@ -0,0 +1,288 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import React from 'react';
+import { InputBoxComponent } from './InputBoxComponent';
+import { render, screen, waitFor, act, fireEvent } from '@testing-library/react';
+// /* @conditional-compile-remove(mention) */
+import userEvent from '@testing-library/user-event';
+// /* @conditional-compile-remove(mention) */
+import { Mention } from './MentionPopover';
+
+describe('InputBoxComponent should call onChange', () => {
+  test('onChange callback should return correct value', async () => {
+    let changedValue = '';
+    render(
+      <InputBoxComponent
+        inlineChildren={true}
+        textValue={changedValue}
+        placeholderText="Enter a message"
+        onChange={(_, newValue) => {
+          changedValue = newValue ?? '';
+        }}
+        maxLength={1000}
+      >
+        <div></div>
+      </InputBoxComponent>
+    );
+    const value = 'Input Box test';
+    // Find the input field
+    const input = await screen.findByPlaceholderText('Enter a message');
+    act(() => {
+      fireEvent.change(input, { target: { value: value } });
+    });
+    await waitFor(async () => {
+      expect(changedValue).toBe(value);
+    });
+  });
+});
+
+/* @conditional-compile-remove(mention) */
+describe('InputBoxComponent should show mention popover', () => {
+  const suggestions: Mention[] = [
+    {
+      id: '1',
+      displayText: 'Test User1'
+    },
+    {
+      id: 'everyone',
+      displayText: 'Everyone'
+    }
+  ];
+  const trigger = '@';
```

Comment:
Maybe add a test for trigger with other character that is not '@' or with trigger that is more than 1 character eg '@@'

Reply:
Added!